xil_carma_cloud/SubsystemControllerParams.yaml

```diff
@@ -46,17 +49,32 @@ guidance:
         - /guidance/plugins/route_following_plugin
         - /guidance/plugins/pure_pursuit_wrapper
         - /guidance/plugins/inlanecruising_plugin
-        - /guidance/plugins/cooperative_lanechange
-      auto_activated_plugins: # TODO: Current logic requires that at most one control be activated at a time. This will be addressed under CAR-6064. https://usdot-carma.atlassian.net/browse/CAR-6064
+      #  - /guidance/plugins/cooperative_lanechange
```

Comment:
Is this supposed to be commented out? Same comment applies to other files as well for `/guidance/plugins/cooperative_lanechange` under `required_plugins`; it looks like it was not commented out before (looking at the `carma-system-4.5.0` release tag)

Reply:
Fixed - updated to use the vru scenario values https://github.com/usdot-fhwa-stol/carma-config/blob/carma-system-4.5.0/ail_vru_uc1_scenario/SubsystemControllerParams.yaml